tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "ES2017",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
```

Consider adding `noImplicitAny` option for better typing:

```suggestion
    "strict": true,
    "noImplicitAny": true,
```

---

I read the "strict"  option already includes this. Isn't it?